src/app/[chainId]/[safeAddress]/[orderId]/page.tsx

```diff
@@ -12,6 +12,9 @@ export default function OrderPage({
     orderId: string;
   };
 }) {
+  if (params.chainId != 11155111) {
```

Comment:
👀

Reply:
For now I didn't deployed the new contract on other chains, so I am disable other chains